packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/EnterAmount/Checkout/validation.tsx

```diff
@@ -99,10 +99,11 @@ export const getMaxMin = (
           // we need minimum of all max amounts including limits
           let limitMaxAmount = Number(pair.buyMax)
           if (limits?.maxOrder) {
-            const baseMaxLimitAmount = Number(
-              convertBaseToStandard('FIAT', limits.maxOrder, false)
+            const buyMaxItem = Number(
+              convertBaseToStandard('FIAT', limitMaxAmount)
```

Comment:
this came back from pairs call and we have to convert it to proper fiat value, main thin here is that we use data from 2 diff endpoints and they are not in same base standard.